core/iwasm/libraries/libc-wasi/sandboxed-system-primitives/src/posix.c

```diff
@@ -172,6 +172,10 @@ convert_errno(int error)
 #undef X
     return code;
 }
+#ifndef BH_PLATFORM_WINDOWS
+bool
+convert_clockid(__wasi_clockid_t in, clockid_t *out);
```

Comment:
worth adding a TODO here and a brief explanation - I presume we only keep it here so poll_oneoff compiles. Because we plan to refactor poll_oneoff, we'll need to remove it as part of that change.